pkg/services/ngalert/api/api_provisioning.go

```diff
@@ -360,3 +426,26 @@ func determineProvenance(ctx *models.ReqContext) alerting_models.Provenance {
 	}
 	return alerting_models.ProvenanceAPI
 }
+
+func exportResponse(c *models.ReqContext, body any) response.Response {
+	format := "json"
+	acceptHeader := c.Req.Header.Get("Accept")
+	if strings.Contains(acceptHeader, "yaml") && !strings.Contains(acceptHeader, "json") {
```

Comment:
These can be comma separated, although I appreciate we might not want to do full parsing of the `Accept` header https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept

Reply:
I'm not sure I follow. If you mean that the Accept header can contain both, then I agree it's up to us to decide what to do here. I chose to only export `yaml` if `json` was not present but I'm open to ideas here.